### PR TITLE
doc: update readme.md

### LINKED ISSUE #17: Update the README of the repository
I propose to update the README.md file for the repository with the following information:
- link to the overview of the organization
- purpose of the repository
- is it in active development or not, is it archived or not, any reasons for that
- links to relevant documentation (security audits, web site, design documents, etc.)

### PR DESCRIPTION
**Pull request by yizha1**
Resolve #17 

Signed-off-by: Yi Zha <yizha1@microsoft.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
-# Archive of Meeting Notes from Hackmd.io
+# Meeting Notes
 
-Active notes are captured at [Notary Project Meeting Notes](https://hackmd.io/_vrqBGAOSUC_VWvFzWruZw?view)
+This repository is in active development and contains archived meeting notes for Notary Project [community meetings](https://notaryproject.dev/community/#community-meetings).
```
[the remark this answers is not in the document]
Comment (FeynmanZhou) [reply README.md:3]: This is not a code repo so I think the word `maintenance` is much more suitable than `development`.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
-# Archive of Meeting Notes from Hackmd.io
+# Meeting Notes
 
-Active notes are captured at [Notary Project Meeting Notes](https://hackmd.io/_vrqBGAOSUC_VWvFzWruZw?view)
+This repository is in active maintenance and contains archived meeting notes for Notary Project [community meetings](https://notaryproject.dev/community/#community-meetings).
```
Comment (toddysm) [inline README.md:3]: I would suggest changing the sentence to:
> This repository is in active maintenance and contains the archive of the meeting notes for Notary Project [community meetings](https://notaryproject.dev/community/#community-meetings).

File: README.md [hunk 3]
```diff
@@ -1,8 +1,12 @@
...
 
-To avoid Hackmd having to index historical content, yearly archives will be moved here.
+Please see the Notary Project [README](https://github.com/notaryproject/.github/blob/main/README.md) file to learn about overall Notary Project.
```
Comment (toddysm) [inline README.md:5]: I would suggest changing to:
> Please see the [Notary Project Overview](https://github.com/notaryproject/.github/blob/main/README.md) to learn about Notary Project.

File: README.md [hunk 4]
```diff
@@ -1,8 +1,12 @@
...
 
-Active notes are captured at [Notary Project Meeting Notes](https://hackmd.io/_vrqBGAOSUC_VWvFzWruZw?view)
+This repository is in active maintenance and contains archived meeting notes for Notary Project [community meetings](https://notaryproject.dev/community/#community-meetings).
 
-To avoid Hackmd having to index historical content, yearly archives will be moved here.
+Please see the Notary Project [README](https://github.com/notaryproject/.github/blob/main/README.md) file to learn about overall Notary Project. 
+
+Active meeting notes are captured at [Notary Project Meeting Notes](https://hackmd.io/_vrqBGAOSUC_VWvFzWruZw?view)
+
+To avoid Hackmd having to index historical content, yearly archives will be stored here.
```
Comment (toddysm) [inline README.md:9]: I would suggest changing to:
> Meeting notes are archived on annual basis in this repository.

**Review by toddysm — CHANGES_REQUESTED** [review]

**Comment by yizha1** [comment]
@toddysm @iamsamirzon @FeynmanZhou I have resolved your comments, please review this PR again. Thanks.

**Review by iamsamirzon — APPROVED** [review]

**Review by priteshbandi — APPROVED** [review]
LGTM

**Review by toddysm — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -1,8 +1,12 @@
-# Archive of Meeting Notes from Hackmd.io
+# Meeting Notes
 
-Active notes are captured at [Notary Project Meeting Notes](https://hackmd.io/_vrqBGAOSUC_VWvFzWruZw?view)
+This repository is in active maintenance and contains the archive of meeting notes for Notary Project [community meetings](https://notaryproject.dev/community/#community-meetings).
```
Comment (SteveLasker) [inline README.md:3]: nit:
...for _the_ Notary Project

**Review by SteveLasker — APPROVED** [review]
LGTM

File: README.md (hunk 5 above)
Comment (yizha1) [reply README.md:3]: Thanks @SteveLasker , I will create a new issue/pr to fix it, see https://github.com/notaryproject/meeting-notes/issues/19

**Pull request merged**